kadi/commands/validate.py

```diff
@@ -718,6 +720,77 @@ def state_vals(self):
         return state_vals_raw
 
 
+class ValidateACISStatePower(ValidateSingleMsid):
+    state_name = "dpa_power"
+    msids = ["dpa_power"]
+    state_keys_extra = ["ccd_count", "clocking", "feps", "ccds", "fep_count", "si_mode"]
+    plot_attrs = PlotAttrs(title="DPA Power", ylabel="DPA Power (W)")
+    min_violation_duration = 600.0  # seconds
+    max_delta_val = 2.0  # W
+
+    def __init__(self, stop=None, days: float = 14, no_exclude: bool = False):
+        from joblib import load
+
+        model_path = Path(os.environ["SKA"]) / "data/kadi/dpa_power_model.joblib"
```

Comment:
Is this too big to be bundled with the package?

Reply:
But in the meantime can you put this file somewhere so I can test this?

Reply:
And that's some of the reason I was wondering about versioning - if running create_dpa_power_model.py with whatever set of parameters were used for the reference joblib will create the same joblib and if we're supposed to do that in test

Reply:
Makes sense - or would it belong in chandra_models?

Reply:
@taldcroft where should I put the file if we bundle it?

Reply:
Putting into `kadi/commands/data` would be fine. This will be a new directory, and you need to add it to the `setup.py`.

Reply:
done